4.analyze-components/scripts/nbconverted/5.visualize-svcca.r

```diff
@@ -9,8 +9,11 @@ svcca_df <- readr::read_tsv(svcca_file,
                             col_types = readr::cols(
                                 .default = readr::col_character(),
                                 svcca_mean_similarity = readr::col_double()))
+print(dim(svcca_df))
 head(svcca_df, 2)
 
```

Comment:
if you're printing the dimensions I would add a string of what's being printed

Reply:
this is being run in a jupyter notebook, so the print statement should be apparent to someone viewing the notebook